addons/website_event_track/views/event_track_templates_proposal.xml

```diff
@@ -76,7 +76,7 @@
                                 <div class="row mb-3">
                                     <label class="col-form-label col-sm-auto" for="tags" style="width: 200px">Categories</label>
                                     <div class="col-sm o_website_event_track_proposal_form_tags">
-                                        <input name="tags" type="text" class="form-control o_wetrack_select2_tags"/>
+                                        <input name="tags" type="text" class="form-control o_wetrack_select_tags"/>
```

Comment:
props.element seems weird indeed. I would try to remove that input here and make your widget "be" the SelectMenu (if this makes sense).

Reply:
you can use a directive t-tag if needed